tests/FakeItEasy.IntegrationTests.External.netstd/copy_output.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+SETLOCAL
+
+SET Configuration=%1
+SET Framework=%2
+
+SET SourceDir=bin\%Configuration%\%Framework%\FakeItEasy.IntegrationTests.External.*
+SET TargetDir=..\FakeItEasy.IntegrationTests.netstd\bin\%Configuration%\netcoreapp1.0
+echo Copying output from %SourceDir% to %TargetDir%
+
+if not exist %TargetDir% mkdir %TargetDir%
+copy /Y %SourceDir%  %TargetDir%

```

Comment:
I'm not sure I understand what you're suggesting. What does `IClassFixture<T>` do?